src/Management/src/EndpointBase/Info/Contributor/GitInfoContributor.cs

```diff
@@ -80,20 +78,5 @@ public virtual IConfiguration ReadGitProperties(string propFile)
 
             return null;
         }
-
-        protected override void AddKeyValue(Dictionary<string, object> dict, string key, string value)
-        {
-            var keyToInsert = key;
```

Comment:
How about if we bring this method back and have it look more like this instead?
```csharp
            object valueToInsert = value;

            if ("time".Equals(key))
            {
                valueToInsert = DateTime.Parse(value).ToString("yyyy-MM-ddTHH:mm:ssZ");
            }

            dict[key] = valueToInsert;
```

Reply:
That's ok, but I have some thoughts:
- I would move the key "time" into a constant (or I would keep it similar to Spring implementation)
- I would define an array of strings as constants instead of a single string (
- I would make sure that DateTime.Parse is culture invariant to avoid incorrect parsing (the values should always be ISO8601 compliant I hope)

Reply:
Sure, that all sounds good. I think it would be fine to use a single constant rather than an array, but if you want to define it as an array that's not likely to hurt

Reply:
I've implemented the changes we discussed. Let me know if you have any further comment.